### PR TITLE
Corrected reference to Azure Active Directory DC

### PR DESCRIPTION
**Pull request by SteveBurkettNZ**
Corrected incorrect reference to Azure Active Directory DC, slight rephrase of sentence, formatting fixes.

### CODE REVIEW

**Review by illfated — APPROVED** [review]
Looks good to me.

File: windows/security/threat-protection/windows-firewall/best-practices-configuring.md [hunk 1]
```diff
@@ -44,13 +42,11 @@ When you open the Windows Defender Firewall for the first time, you can see the
 
 *Figure 1: Windows Defender Firewall*
 
-1.  **Domain profile**: Used for networks where there is a system of account authentication against a domain controller (DC), such as an Azure Active Directory DC
+1.  **Domain profile**: Used for networks where there is a system of account authentication against an Active Directory Directory Services domain controller, such as a corporate network.
```
Comment (schmurky) [inline windows/security/threat-protection/windows-firewall/best-practices-configuring.md:45]: an Active Directory **Domain** Services controller

**Review by schmurky — CHANGES_REQUESTED** [review]

File: windows/security/threat-protection/windows-firewall/best-practices-configuring.md (hunk 1 above)
Comment (illfated) [reply windows/security/threat-protection/windows-firewall/best-practices-configuring.md:45]: Fair point, it does not exactly sit well on the tongue to read it out loud using "Directory Directory", but on the other hand, "Domain Services domain controller" does not seem correct either.

**Comment by e0i** [comment]
Please feel free to re-open this PR to confirm and apply the requested changes. Thank you. @SteveBurkettNZ

**Pull request closed without merging**